### PR TITLE
Test metrics enable and then disable

### PR DESCRIPTION
**Pull request by mkemel**
Adds an integration test which checks that metrics are enabled and disabled correctly back-and-forth

Relates to #668

### CODE REVIEW

File: tests/tests/tier0/metrics-enable-and-disable/test_metrics_enable_and_disable.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    target_dir = os.path.join("/", "etc", "systemd", "system")
+    foo.copy_systemd_service(simple_service, source_dir, target_dir)
+    assert foo.wait_for_unit_state_to_be(simple_service, "inactive")
+
+    result, output = ctrl.run_python(os.path.join("python", "metrics_is_enabled.py"))
+    if result == 0:
+        raise Exception(f"Metrics not disabled: {output}")
+    assert foo.wait_for_unit_state_to_be(simple_service, "inactive")
+
+    ctrl.bluechictl.enable_metrics(True, 0)
```
Comment (engelmi) [inline tests/tests/tier0/metrics-enable-and-disable/test_metrics_enable_and_disable.py]: NIT: `enable_metrics` is called with the default values, so those can be omitted here.

File: tests/tests/tier0/metrics-enable-and-disable/test_metrics_enable_and_disable.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+    result, output = ctrl.run_python(os.path.join("python", "metrics_is_enabled.py"))
+    if result != 0:
+        raise Exception(f"Metrics not enabled: {output}")
+    assert foo.wait_for_unit_state_to_be(simple_service, "inactive")
+
+    ctrl.bluechictl.disable_metrics(True, 0)
```
Comment (engelmi) [inline tests/tests/tier0/metrics-enable-and-disable/test_metrics_enable_and_disable.py]: NIT: `disable_metrics` is called with the default values, so those can be omitted here.

**Review by engelmi — APPROVED** [review]
2 NITs, otherwise LGTM

File: tests/tests/tier0/metrics-enable-and-disable/test_metrics_enable_and_disable.py (hunk 2 above)
Comment (mkemel) [reply tests/tests/tier0/metrics-enable-and-disable/test_metrics_enable_and_disable.py]: Done

File: tests/tests/tier0/metrics-enable-and-disable/test_metrics_enable_and_disable.py (hunk 1 above)
Comment (mkemel) [reply tests/tests/tier0/metrics-enable-and-disable/test_metrics_enable_and_disable.py]: Done

**Pull request merged**